src/tests/test_crypto.py

```diff
@@ -76,38 +59,16 @@ def _hashdigest(data):
         return RIPEMD.RIPEMD160Hash(data).digest()
 
 
-class TestAddresses(unittest.TestCase):
-    """Test addresses manipulations"""
-    def test_base10_multiply(self):
-        """Test arithmetic.base10_multiply"""
-        self.assertEqual(
-            sample_point,
-            arithmetic.base10_multiply(arithmetic.G, sample_factor))
+class TestHighlevelcrypto(unittest.TestCase):
+    """Test highlevelcrypto public functions"""
 
     def test_privtopub(self):
         """Generate public keys and check the result"""
         self.assertEqual(
-            arithmetic.privtopub(sample_privsigningkey).encode(),
+            highlevelcrypto.privToPub(sample_privsigningkey),
             hexlify(sample_pubsigningkey)
         )
         self.assertEqual(
-            arithmetic.privtopub(sample_privencryptionkey).encode(),
+            highlevelcrypto.privToPub(sample_privencryptionkey),
             hexlify(sample_pubencryptionkey)
         )
-
-    def test_address(self):
-        """Create address and check the result"""
-        from pybitmessage import addresses
-        from pybitmessage.fallback import RIPEMD160Hash
-
-        sha = hashlib.new('sha512')
-        sha.update(sample_pubsigningkey + sample_pubencryptionkey)
-        ripe_hash = RIPEMD160Hash(sha.digest()).digest()
-        self.assertEqual(ripe_hash, unhexlify(sample_ripe))
```

Comment:
I think this test is missing in the new code.

Reply:
This part looks like `TestCrypto.test_hash_string()` above. The rest part is in `TestAddresses` `.test_decode()` and `test_encode()`.

Reply:
Ok I missed that, looks ok.